src/DOM.js

```diff
@@ -471,7 +490,7 @@ class Element extends Node {
 
     this.on('attribute', (name, value) => {
       if (name === 'id') {
-        if (this.ownerDocument.defaultView[value] === undefined) {
+        if (this.ownerDocument && this.ownerDocument.defaultView[value] === undefined) {
```

Comment:
I'm not sure this is the right way to go, because having `ownerDocument` looks [non-optional](https://www.w3.org/TR/dom/#dom-node-ownerdocument) for any `Node`.

I think the reason this crashes is because of the way the test constructs the `Element`.

If we use `document.createElement` instead I think the problem goes away.